src/clojure/nrepl/core.clj

```diff
@@ -228,6 +228,23 @@
   (throw (IllegalArgumentException.
           (format "No nREPL support known for scheme %s, url %s" (uri-scheme uri) uri))))
 
+(defmulti url-scheme
```

Comment:
Funny enough there's already a function named `url-scheme` in this namespace, so we either have to make it more generic or perhaps rename the other function, which is private.

Reply:
There is a `uri-scheme`function in this ns, but I think I should change the name of `url-scheme` to something else, like `transport->uri-scheme` so it says better what it's about